pkg/config/sampling/sampling.go

```diff
@@ -77,10 +82,42 @@ func (u *Config) Get() *corev1.ConfigMap {
 	}
 }
 
+// CheckForSamplingConfigFile will check if there is a config file present
+// if there is one it returns true
+func CheckForSamplingConfigFile(jaeger *v1.Jaeger) bool {
+	options := v1.Options{}
+
+	// check for deployment strategy
+	if jaeger.Spec.Strategy == v1.DeploymentStrategyAllInOne {
+		options = jaeger.Spec.AllInOne.Options
+	} else {
+		options = jaeger.Spec.Collector.Options
+	}
+
+	jsonObject, err := options.MarshalJSON()
```

Comment:
Sorry should have picked up on this earlier, but there is a more direct way to test if the option has been defined when using the options, e.g.
```
if _, exists := options["sampling.strategies-file"]; exists {
     ....
     return true
}
```